### PR TITLE
Variance of Laplacian

### PR DESCRIPTION
**Pull request by Saurav-D**
# Peer Review Information

Added code for calculating:
- Laplacian of Image
- Variance of laplacian of image
- Var of lap of background on foreground based on the bounding box information.
- Var of lap of background on foreground based on the segmentation information.
- Added notebook for visualization

# Pull Request Check List

- [x] Added **tests** for changed code.
- [x] Updated **documentation** for changed code.

### CODE REVIEW

File: datasetinsights/stats/image_analysis/laplacian.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        np.ndarray: numpy array of Laplacian of the image
+    """
+    image = cv2.imread(img_path)
+    gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+    laplacian = cv2.Laplacian(gray, cv2.CV_64F)
+    laplacian = laplacian.astype("float")
+    return laplacian
+
+
+def get_img_var_laplacian(img_path: str) -> np.ndarray:
```
Comment (davidwang-unity) [inline datasetinsights/stats/image_analysis/laplacian.py]: this function may be an overkill here, compared `get_img_var_laplacian(img_path)` with `laplacian_img(img_path).var()`

File: datasetinsights/stats/image_analysis/laplacian.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        img_path (str): Path of image
+
+    Returns:
+        Variance of Laplacian of image
+    """
+    return laplacian_img(img_path).var()
+
+
+def get_bbox_var_laplacian(
+    laplacian: np.ndarray, x: float, y: float, w: float, h: float
```
Comment (davidwang-unity) [inline datasetinsights/stats/image_analysis/laplacian.py]: why set `x y w h` as `float` here if we need to transfer them as `int` in the function?

File: datasetinsights/stats/image_analysis/laplacian.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        image
+        img_var_laplacian (np.ndarray): Variance of Laplacian of background
+        of the image
+
+    """
+    bbox_var_lap = []
+    img_laplacian = laplacian
+
+    for ann in annotations:
+        x, y, w, h = ann["bbox"]
```
Comment (davidwang-unity) [inline datasetinsights/stats/image_analysis/laplacian.py:64]: maybe we can directly transfer `x y w h` as `int` type here?

File: datasetinsights/stats/image_analysis/laplacian.py (hunk 1 above)
Comment (Saurav-D) [reply datasetinsights/stats/image_analysis/laplacian.py]: Removed

File: datasetinsights/stats/image_analysis/laplacian.py (hunk 2 above)
Comment (Saurav-D) [reply datasetinsights/stats/image_analysis/laplacian.py]: Fixed

**Review by davidwang-unity — APPROVED** [review]
LGTM!

**Pull request merged**